css/style.css

```diff
@@ -8,656 +8,1472 @@
-  color: #253858;
-  justify-items: center;
-  border-radius: 8px;
-  background-color: #f7f7f9;
-}
-
-.works-frame {
-  /* border: 1px solid black; */
-  margin-top: -70px;
-  width: 100%;
-  height: 250px;
-  margin-left: -4%;
-  padding: 10px;
-}
-
-.display-skills {
-  border: 1px solid black;
-  list-style: none;
-}
-
-.frame-work {
-  list-style: none;
-  display: flex;
-  gap: 25px;
-  color: #000;
-  order: 1;
-  margin: 35px;
-  margin-top: 70px;
-  align-items: center;
-  margin-left: -2%;
-  font-weight: 500;
-  font-size: 20px;
-  line-height: 24px;
-}
-
-.left-arrow {
-  display: flex;
-  margin-left: 55%;
-  left: 41.67%;
-  right: 33.33%;
-  top: 25%;
-  bottom: 2;
-}
-
-.left-arrow2 {
-  display: flex;
-  margin-left: 80%;
-  left: 41.67%;
-  right: 33.33%;
-  top: 25%;
-  bottom: 2;
-}
-
-.seperator-btn {
-  width: 327px;
-  height: 1px;
-  left: 0;
-  right: 0;
-  bottom: 0;
-  background: #dfe1e6;
-}
-
-.contanct-containter {
-  display: flex;
-  margin-top: 60%;
-  max-width: 365px;
-  margin-left: -5%;
-  background-color: #6070ff;
-  border-top-left-radius: 100px;
-}
-
-.contact-info {
-  border-top-left-radius: 100px;
-  top: 5410px;
-  height: 100%;
-}
-
-.backgound-image {
-  background-image: url(/images/contact-form-shapes-mobile.svg);
-  background-repeat: no-repeat;
-  object-fit: cover;
-  margin-top: 2%;
-  position: absolute;
-  width: 265px;
-  height: 709.17px;
-  margin-left: 142px;
-}
-
-.contact-title {
-  margin-top: 30%;
-}
-
-.title2 {
-  text-align: center;
-  font-weight: 700;
-  font-size: 40px;
-  line-height: 52px;
-  color: #fff;
-  margin-left: -1%;
-}
-
-.contacts-info {
-  font-style: normal;
-  font-weight: 400;
-  font-size: 20px;
-  line-height: 28px;
-  color: #ebebff;
-  display: flex;
-  padding: 0 13%;
-}
-
-.action-form {
-  display: flex;
-  flex-direction: column;
-  row-gap: 11px;
-}
-
-.contact-form {
-  border-radius: 5px;
-  padding: 0 20px;
-}
-
-input[type=text],
-input[type=email],
-select,
-textarea {
-  width: 100%;
-  padding: 5px;
   box-sizing: border-box;
-  margin-top: 4px;
-  margin-bottom: 16px;
-  border-radius: 10px;
-  border: 1px solid #cfd8dc;
-  outline: none;
-}
-
-.text-area {
-  height: 170px;
-  color: #b3bac5;
-}
-
-.label::placeholder {
-  color: #172b4d;
-}
-
-.label {
-  outline: none;
-  width: 327px;
-  height: 48px;
-  border: 1px solid #cfd8dc;
-  border-radius: 8px;
-  left: 16px;
-}
-
-.submit-btn2 {
-  display: flex;
-  align-items: center;
-  padding: 12px 18px 12px 18px;
-  width: 140px;
-  height: 48px;
-  left: 470px;
-  top: 5470px;
-  background: #fff;
-  border-radius: 8px;
-  border: none;
-  color: #6070ff;
-  font-size: 17px;
-  margin-right: 0;
-}
-
-.img-form {
-  margin: 0 auto;
-  display: flex;
-  padding: 7%;
 }
 
-form {
-  margin-top: 2%;
+@media (min-width: 480px) {
+  html,
+  body {
+    width: 100%;
+    height: 100%;
+    overflow-x: hidden;
+    padding: 0;
+    margin: 0;
+  }
+
+  .logo-part {
+    display: none;
+  }
```

Comment:
- Kindly, be sure that your headline part sticks to the [Figma design](figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1&t=xmWXLXW7NiSVxIum-0), you missed the background image and header padding, please fix it👌👍


| Your design                                                             | Figma design                                         |
| --------------------------------------------------------------------------------| --------------------------------------------------------- |
|<img src="https://user-images.githubusercontent.com/52010789/206550756-a0fdc0ca-4764-4bac-8c52-d52f4440b0e9.png" width="200">|<img src="https://user-images.githubusercontent.com/52010789/206550781-55996082-1f0e-4d79-959a-6960d05ad217.png" width="200">|


😎️